tools/Microsoft.TemplateEngine.Authoring.TemplateVerifier/VerificationEngine.cs

```diff
@@ -234,6 +234,12 @@ await VerifyResult(options, commandResult, string.IsNullOrEmpty(sourceFile) ? st
                     string relativePath = fileSystem.PathRelativeTo(filePath, contentDir);
                     return includeGlobs.Any(g => g.IsMatch(relativePath)) && !excludeGlobs.Any(g => g.IsMatch(relativePath));
                 },
+                options: new EnumerationOptions()
+                {
+                    RecurseSubdirectories = true,
+                    // Workarounding Verify bug https://github.com/VerifyTests/Verify/issues/699
+                    AttributesToSkip = 0
```

Comment:
just curious, what does this option do?

Reply:
It instructs `Directory.EnumerateFiles` not to skip any files. It's a default behavior of `Directory.EnumerateFiles` if the overload with `SearchOptions` is being used while passing `SearchOption.AllDirectories`

[`Directory.EnumerateFiles`](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/IO/Directory.cs#L232) --> [`EnumerationOptions.FromSearchOption`](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/IO/EnumerationOptions.cs#L47) --> [`EnumerationOptions.CompatibleRecursive`](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/IO/EnumerationOptions.cs#L24)

Since `Verify` is using the overload with `SearchOptions` to discover existing snapshot files, but overload with `EnumerationOptions` to discover target files, we need to make sure to use exact same way of discovering the target files (in our case the generated template files) - otherwise we'll be cought in a situation when `Verify.VerifyDirectory` sees some snapshots as extra ones, since it ignores the target files.